### PR TITLE
Adds Image faker

### PR DESCRIPTION
**Pull request by igorgoncalves**
## Connection with issue(s)

Resolve issue #19

## Testing and Review Notes
 Generates a url to a image random image. The size of image is determined  by `width` and `height` paramters. You can choose the image categories related  by the `keywords` parameter separeted by comma .
 
Example:
 ```dart
    faker.image.image(width: 1200, height: 900, keywords: 'nature,people');
 ```
Result: https://source.unsplash.com/1200x900?nature,people

## To Do

- [X] double check the original issue to confirm it is fully satisfied
- [X] add testing notes and screenshots in PR description to help guide reviewers

### CODE REVIEW

File: lib/src/image.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class Image {
+  const Image();
+
+  /// Generates a url to a image random image. The size of image is determined
+  /// by `width` and `height` paramters. You can choose the image categories related
```
Comment (drager) [inline lib/src/image.dart]:
```suggestion
  /// by `width` and `height` parameters. You can choose the image categories related
```

**Comment by drager** [comment]
@igorgoncalves Thanks a lot! I had a thought that instead of passing a comma separated string for the keywords we could actually accept a List of strings e.g: `faker.image.image(width: 1200, height: 900, keywords: ['nature', 'people']);`. What do you think?

**Comment by igorgoncalves** [comment]
Make sense to me, I will change that :D

**Comment by drager** [comment]
Great, thanks!

**Comment by igorgoncalves** [comment]
The tests fails in Travis because this #22  I create a PR to resolve  #23

File: lib/src/image.dart [hunk 2]
```diff
@@ -19,19 +19,19 @@ class Image {
...
     var url = 'https://source.unsplash.com';
     
     url += '/${width}x${height}';
 
     if (keywords.isNotEmpty) {
       var keywordFormat =
-          new RegExp(r'^([A-Za-z0-9].+,[A-Za-z0-9]+)$|^([A-Za-z0-9]+)$');
-      if (keywordFormat.hasMatch(keywords)) {
-        url += '?' + keywords;
+          new RegExp(r'^([A-Za-z0-9].+,[A-Za-z0-9]+)$|^([A-Za-z0-9]+)$');
```
Comment (drager) [inline lib/src/image.dart:29]: Please run dart formatter on this file.

**Review by drager — APPROVED** [review]
Thanks a lot! It looks good!

**Pull request merged**